lib/tasks/package.js

```diff
@@ -25,6 +26,8 @@ class PackageTask extends Task {
     } else {
       // Assemble the project and pass to forge
       projectPath = quickTemp.makeOrRemake(this, '-tmpPath');
+      childProcess.execSync(`chmod a+x ${projectPath}`);
```

Comment:
Sg, but refactoring away from `quickTemp` is definitely out of scope for this change. Maybe let's check for windows, or otherwise catch a failed run?